internal/cri/server/restart.go

```diff
@@ -477,7 +477,7 @@ func cleanupOrphanedIDDirs(ctx context.Context, cntrs []containerd.Container, ba
 
 func (c *criService) createContainerIO(containerID, sandboxID string, config *runtime.ContainerConfig) (*cio.ContainerIO, error) {
 	if config == nil {
-		return nil, fmt.Errorf("ContainerConfig should not be nil when create container io")
+		return nil, fmt.Errorf("containerConfig should not be nil when create container io")
```

Comment:
I think we should just keep the uppercase and ignore the lint here as `ContainerConfig` is a valid type.